.changelog/release-v14.1.0.md

```diff
@@ -90,7 +96,7 @@ Date | Revision | Description
 
 ## 6. Deprecations
 
-The following components are being deprecated from the packaged Mojaloop Helm chart release:
+The following components are being deprecated from the packaged Mojaloop Helm chart release and will likely be replaced with [Business Operation Framework (BOF)](https://github.com/mojaloop/business-operations-framework-docs) in the next Mojaloop official release (possibly v15):
```

Comment:
Hey @elnyry-sam-k , we should discuss this.

I do not recommend deploying BoF as part of a Mojaloop Install. The Helm charts are going to be "MASSIVE", and the BoF deployment should be handled seperately like our Backends in v15.

Reply:
```suggestion
The following components are being deprecated from the packaged Mojaloop Helm chart release and will likely be replaced with [Business Operation Framework (BOF)](https://github.com/mojaloop/business-operations-framework-docs) in the a future Mojaloop official release:
```